src/init.cpp

```diff
@@ -1809,6 +1821,11 @@ bool AppInitMain(const util::Ref& context, NodeContext& node, interfaces::BlockA
         GetBlockFilterIndex(filter_type)->Start();
     }
 
+    if (args.GetBoolArg("-coinstatsindex", DEFAULT_COINSTATSINDEX)) {
+        g_coin_stats_index = MakeUnique<CoinStatsIndex>(coin_stats_cache_size, false, fReindex);
```

Comment:
Please [use `std::make_unique` in new code](https://github.com/bitcoin/bitcoin/blob/master/src/util/memory.h#L13).

Reply:
Done